src/scripts.js

```diff
@@ -1,56 +1,64 @@
 const userInfoDisplay = document.querySelector('.user-info__basic-display');
-let userFriendsDisplay = document.querySelector('.user-info__friends')
-
+const currentDate = document.querySelector('.user-info__date-input')
+const dateButton = document.querySelector('.user-info__date-button')
 const hydrationCurrentCount = document.querySelector('.hydration__current');
 const hydrationDate = document.querySelector('.hydration__date');
-let hydrationWeek = document.querySelector('.hydration__week-list');
-
 const sleepCurrentHours = document.querySelector('.sleep__current-hours');
 const sleepCurrentQuality = document.querySelector('.sleep__current-quality');
-let sleepWeekHours = document.querySelector('.sleep__week-hours');
-let sleepWeekQuality = document.querySelector('.sleep__week-quality');
 const sleepAveragesHours = document.querySelector('.sleep__averages-hours');
 const sleepAverageQuality = document.querySelector('.sleep__averages-quality');
-
 const activityCurrentSteps = document.querySelector('.activity__current-steps');
 const activityCurrentMinutes = document.querySelector('.activity__current-minutes');
 const activityCurrentMiles = document.querySelector('.activity__current-miles');
 const activityAverageSteps = document.querySelector('.activity__current-average-steps')
 const activityAverageMinutes = document.querySelector('.activity__current-average-minutes')
 const activityAverageMiles = document.querySelector('.activity__current-average-miles')
-const activityWeekSteps = document.querySelector('.activity__week-steps');
-const activityWeekStairs = document.querySelector('.activity__week-stairs');
-const activityWeekMinutes = document.querySelector('.activity__week-minutes');
 
 const cards = document.querySelectorAll(".card");
 
 function flipCard() {
   this.classList.toggle("flip");
 }
-cards.forEach((card) => card.addEventListener("click", flipCard));
-
-let allUsers = userData.map(userInfo => {
-  const user = new User(userInfo);
-  return user;
-});
 
-let userRepo = new UserRepository(allUsers);
+cards.forEach((card) => card.addEventListener("click", flipCard));
 
+let allUsers
+let userRepo
 let today;
 let currentUser;
 let activityInfo;
 let hydrationInfo;
-let sleepInfo
+let sleepInfo;
+let thisWeeksDates;
+
+const randomUser = () => {
+  return Math.floor(Math.random()*50)
+}
 
 const pageLoad = () => {
-  currentUser = userRepo.userData[25];
+  allUsers = userData.map(userInfo => {
+    const user = new User(userInfo);
+    return user;
+  });
+  userRepo = new UserRepository(allUsers);
+  currentUser = userRepo.userData[randomUser()];
   userFirstName.innerText = `Hello, ${currentUser.returnUserFirstName()}!`;
+  activityInfo = new Activity(activityData);
+  hydrationInfo = new Hydration(hydrationData);
+  sleepInfo = new Sleep(sleepData);
+  today = activityInfo.returnLatestDate();
+  currentDate.value = today.replaceAll("/", "-");
+  generatePageDisplay();
+};
```

Comment:
I think we could refactor this further by moving the instantiations to their own helper function as well, but glad to see we're already doing that with the display!